app/src/main/java/com/framgia/soundclound/screen/detailalbum/DetailAlbumActivity.java

```diff
@@ -0,0 +1,37 @@
+package com.framgia.soundclound.screen.detailalbum;
+
+import android.content.Context;
+import android.content.Intent;
+import android.databinding.DataBindingUtil;
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+
+import com.framgia.soundclound.R;
+import com.framgia.soundclound.databinding.ActivityAlbumDetailBinding;
+import com.framgia.soundclound.screen.detailgenre.GenreDetailActivity;
```

Comment:
ERROR: Unused import: com.framgia.soundclound.screen.detailgenre.GenreDetailActivity. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports